client/src/app/shared/progress-bar/progress-bar.component.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { MatProgressBarModule } from '@angular/material';
+import { ProgressBarComponent } from './progress-bar.component';
+import { ProgressBarService } from './progress-bar.service';
+
+describe('ProgressBarComponent', () => {
+  let component: ProgressBarComponent;
+  let fixture: ComponentFixture<ProgressBarComponent>;
+  let svc: ProgressBarService;
+  beforeEach(
+    async(() => {
+      TestBed.configureTestingModule({
+        imports: [MatProgressBarModule],
+        declarations: [ProgressBarComponent],
+        providers: [ProgressBarService]
+      }).compileComponents();
+    })
+  );
```

Comment:
This LGTM! The only thing I'd change here is that `async()` and `compileComponents()` is not actually needed.

This is code from a time where `compileComponents()` was async due to fetching of templates of components. Ever since CLI switched to webpack and template inlining, `async()` and `compileComponents()` isn't needed anymore. Components are always synchronously compiled on bootstrap :)